Procfile

```diff
@@ -1,4 +1,5 @@
 web: bundle exec puma -C config/puma.rb
+all_workers: bundle exec rake workers:all:run
 message_log_worker: bundle exec rake workers:message_logger:run
```

Comment:
do the other workers still work? if not, we should remove these lines from the Procfile